### PR TITLE
Correcting reference to anchor

### PR DESCRIPTION
**Pull request by darrylhein**

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -755,7 +755,7 @@ Here, we set it to 2 for the ``$child`` property:
 The metadata loader corresponding to the chosen format must be configured in
 order to use this feature. It is done automatically when using the Symfony
 Standard Edition. When using the standalone component, refer to
-:ref:`the groups documentation <component-serializer-attributes-groups>` to
+:ref:`the groups documentation <attributes-groups>` to
```
Comment (xabbuh) [inline components/serializer.rst]: The `attributes-group` target does not exist. Can you explain why you think this change is necessary?

Comment (darrylhein) [inline components/serializer.rst]: Sorry? The current link goes to `http://symfony.com/doc/master/components/serializer.html#component-serializer-attributes-groups` which doesn't exist, but `http://symfony.com/doc/master/components/serializer.html#attributes-groups` does and thus why I changed it to  `attributes-groups`. I've verified this on the version 4 of the docs on the website.

Comment (xabbuh) [inline components/serializer.rst]: There is a `span` element with this id in the docs. But it seems that some JavaScript or the like is breaking the handling of anchors on the documentation website currently. `attributes-groups` anchor that you used here is generated automatically based on the headline. However, reStructuredText does not allow to just use this anchor. To be able to use it in a `ref` role one must define an explicit label.

Comment (darrylhein) [inline components/serializer.rst]: Okay, then I guess this isn't applicable. JS needs to be fixed instead.

**Pull request closed without merging**